src/app/home/home.component.ts

```diff
@@ -110,6 +112,7 @@ export class HomeComponent implements OnInit {
     private route: ActivatedRoute,
     private router: Router,
     @Inject(PLATFORM_ID) private platformId: Object,
+    @Optional() @Inject(RESPONSE) private response: any,
```

Comment:
```suggestion
    @Optional() @Inject(RESPONSE) private response: Response,
```

Requires importing Response, I think this is the right one:
```
import { Response } from 'express';
```